gbpservice/nfp/lib/nfp_log_helper.py

```diff
@@ -0,0 +1,144 @@
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import json
+
+from collections import OrderedDict
+from neutron._i18n import _LE
+from oslo_log import log as logging
+
+LOG = logging.getLogger(__name__)
+
+
+def prepare_log_meta_data(nf=None, log_meta_data=None,
+                          level=None, event_category=None,
+                          event=None, service_type=None,
+                          service_provider=None):
+    """Prepares log meta data string in a specific format
+
+    :param nf - network function object
+    :param log_meta_data - already existing log_meta_data string to restructure
+                           it along with other params like level,event etc
+    :param level - log level
+    :param event_category
+    :param event
```

Comment:
For param level, event_category, event, Can it be any string ?
For level we can define constants.